### ISSUE TITLE
scrollToIndex with "smooth" behavior blanks out list when scrolling large distances

### ISSUE THREAD

**Issue by dilizarov** [petyosi/react-virtuoso#229]
The following video is taken from the virtuoso.dev website

[Video](https://streamable.com/3pj1p8)

I haven't confirmed if this happens over small distances as well.

Environment:

macOS Big Sur Version 11.0.1
Google Chrome Version 87.0.4280.88

**Comment by petyosi** [comment]
Yep, confirming that. Smooth scrolling over large lists is quite an intensive action. Not sure if addressible.

**Comment by dilizarov** [comment]
Yeah, it is understandable why it happens - but it is quite noticeable.

Maybe while considering a fix, it is good to acknowledge that in these cases... people should just use `auto` behavior.

**Comment by dilizarov** [comment]
Also, off-topic but fantastic job with API choices (followOutput and atBottomStateChange) for example.

This library really feels like it could be the future of scalable chat UX on the web.

**Comment by RomanSkrypnik** [comment]
Is it possible to fix at all? I face the same problem (
